### ISSUE TITLE
testing: Add frontend tests.

### ISSUE THREAD

**Issue by aero31aero** [zulip/zulip-js#52]
Currently, we test the code examples in a node runtime. However, zulip-js also functions in the browser, and we should add bundling and testing in a browser based environment as well.

This task is open ended in terms of which bundler(s) to use and which testing framework to use. Ideally, we should be able to have a way to run all the currently existing tests in a stubbed out environment without the node utilities/packages available.

**Comment by vaibhavrajsingh2001** [comment]
@aero31aero is this issue still open/requires work?

**Comment by aero31aero** [comment]
Yes, this is still available. There haven't been any updates on this front.

**Comment by SinghCod3r** [comment]
@aero31aero , hey can you assign this to me. i want to work on this.

**Comment by aero31aero** [comment]
Hey, thanks for deciding to take it up.

We don't need to use the assignment feature in this repo, not at the
moment; your comment above is enough indication that you're working on it
while you prepare a PR.

**Comment by AarishMansur** [comment]
Hey @aero31aero  is this issue still relevant ?? Im planning to extend the current setup to support a browser like environment (e.g., via jsdom or a browser runner) while keeping existing Node tests unchanged.

if this sounds good I will proceed with a  PR
